src/Microsoft.Graph.Core/Requests/Content/BatchRequestContent.cs

```diff
@@ -308,10 +308,10 @@ private string GetHeaderValuesAsString(IEnumerable<string> headerValues)
         private string GetRelativeUrl(Uri requestUri)
         {
             string version = "v1.0";
-            if (requestUri.AbsoluteUri.Contains("beta"))
+            if (requestUri.AbsoluteUri.Contains("/beta/"))//use the url segment to search.
                 version = "beta";
 
-            return requestUri.AbsoluteUri.Substring(requestUri.AbsoluteUri.IndexOf(version) + version.ToCharArray().Count());
+            return requestUri.AbsoluteUri.Substring(requestUri.AbsoluteUri.IndexOf(version, StringComparison.OrdinalIgnoreCase) + version.ToCharArray().Length);
```

Comment:
```suggestion
            return requestUri.PathAndQuery[5..]; // `v1.0/` and `beta/` are both 5 characters
```
And remove everything above

Reply:
Thanks! This is resolved via https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/pull/312/commits/d56ccb1f312b4d63fb48b5fa89ae2b8616456bee below as the Range operators need netstandard 2.1 and above
```cs
        return requestUri.PathAndQuery.Substring(5);
```